encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -25,14 +25,17 @@
 package main
 
 import (
+	"flag"
 	"path/filepath"
 	"strings"
 
 	"go.uber.org/thriftrw/plugin"
 	"go.uber.org/thriftrw/plugin/api"
 )
 
-const serverTemplate = `
+var _uberfx = flag.Bool("uberfx", false, "generate code for UberFx with fx.Context")
```

Comment:
I see. I was actually thinking to not complicate the existing template, since its readable in its current form. passing context, and adding checks would just complicate the template. I have also updated the import and New function in the template. We would need some way to generate the clear code.
 Do you have a an example I can follow to inject the context path in the existing template?